src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -64,6 +66,66 @@
                   "with": "src/environments/environment.prod.ts"
                 }
               ]
+            },
+            "cs": {
+              "outputPath": "dist/cs/",
+              "i18nFile": "src/locale/messages.cs.xlf",
+              "i18nLocale": "cs"
+            },
+            "de-DE": {
+              "outputPath": "dist/de-DE/",
+              "i18nFile": "src/locale/messages.de-DE.xlf",
+              "i18nLocale": "de-DE"
+            },
+            "en-US": {
+              "outputPath": "dist/en-US/",
+              "i18nFile": "src/locale/messages.en-US.xlf",
```

Comment:
@tspmelo can you explain?

Reply:
`ng serve` doesn't support i18n flags, so we can't just use the current `npm start` to serve angular in portuguese or spanish.
If we want to specify any language we need to resort to using configurations and use the `--configuration` flag.
p.e.: `npm start -- --configuration=pt-BR`

Reply:
:see_no_evil: @tspmelo in this case, in order to minimize duplication, would it be better to put everything in angular.json instead of package.json?

Reply:
For the production build of languages we would need to copy [angular.json#L51](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/angular.json#L51 ) each language. So it would have lots of repeated lines and would not reduce the total lines of code.

Besides that, we would stop being able to use the `npm-run-all` and would have revert the make file to use the previous language iteration code.